### DISCUSSION TITLE
Erro ao Baixar Video

### DISCUSSION THREAD

**Discussion by marianafernandes2204** [aclap-dev/video-downloadhelper#521] [Help and Q&A]
### Your question

Estou a dias tentando baixar alguns videos e nao permite, sempre com o mesmo erro:
{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://app.nutror.com/curso/b1921949830e64e2a56905764543ac00965c5172/aula/4950214",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 8 times\r\n[in#0 @ 0000029f3c898e00] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJhOTA0YTI1My1jZGI3LTQ0YmMtOGYyNC04NWY4YmIxMzhlZDUiLCJzZXNzaW9uS2V5IjoidmtteXBrcnMiLCJwcm9kdWNlcktleSI6IjYwNTQ2NCIsInByb2R1Y2VyRW1haWwiOiJhcmlhbmUuY2FtcG9saW1AZ21haWwuY29tIiwiY29uc3VtZXJFbWFpbCI6Im1hcmlhbmFjbmV2ZXNAaG90bWFpbC5jb20iLCJpbXByaW50IjpmYWxzZSwic3RvcmFnZSI6InIyIiwicGwiOiIxMDgwcF9wbGF5bGlzdC5tM3U4IiwiaWF0IjoxNzE2MjM5MzcyLCJleHAiOjE3MTY0MTIxNzJ9.ytIn4-1NDrTBbL0iIJocLOx8I61eKk7j_NDyheo886M.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2770956744849520",
    "error": "coapp_failure",
    "id": "coapp_failure_5f3ed470-c25d-4c87-a794-dbccb5af744c"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://app.nutror.com/curso/b1921949830e64e2a56905764543ac00965c5172/aula/4950214",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 8 times\r\n[in#0 @ 0000029f3c898e00] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJhOTA0YTI1My1jZGI3LTQ0YmMtOGYyNC04NWY4YmIxMzhlZDUiLCJzZXNzaW9uS2V5IjoidmtteXBrcnMiLCJwcm9kdWNlcktleSI6IjYwNTQ2NCIsInByb2R1Y2VyRW1haWwiOiJhcmlhbmUuY2FtcG9saW1AZ21haWwuY29tIiwiY29uc3VtZXJFbWFpbCI6Im1hcmlhbmFjbmV2ZXNAaG90bWFpbC5jb20iLCJpbXByaW50IjpmYWxzZSwic3RvcmFnZSI6InIyIiwicGwiOiIxMDgwcF9wbGF5bGlzdC5tM3U4IiwiaWF0IjoxNzE2MjM5MzcyLCJleHAiOjE3MTY0MTIxNzJ9.ytIn4-1NDrTBbL0iIJocLOx8I61eKk7j_NDyheo886M.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2770956744849520",
    "error": "coapp_failure",
    "id": "coapp_failure_5f3ed470-c25d-4c87-a794-dbccb5af744c"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

The URL of the web page in your error message goes to a site that requires a login.  I am not going to register for any site that requires a login just to diagnose a user problem.

The other URL in your error message appears to be for a manifest.  It turns out I was able to retrieve that file.  It was not in a standard format.  All of the data was wrapped around so it was a long file of a single line.  However, I recognized that this was actually a stream manifest for something, & the URLs in it were complete, not partial.  So, using my text editor to unmangle the file, I created what I hoped would be a proper manifest on my system.  I used that to try to download that stream using ffmpeg.  Unfortunately, ffmpeg generated an error saying it had encountered an empty segment.  I suspect the URLs in this manifest have expired.

But the problem seems to be that the manifest is not in the correct format to be a manifest.  I don't know what VDH could do in such a case.  Maybe @paulrouget Paul can offer greater insights.  He actually writes the VDH code.

**Comment by marianafernandes2204** [comment]
olá, tentei no mesmo site um curso diferente e apareceu novo erro:
{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://app.nutror.com/curso/6f8e3a83c5127d7195581e541ba5b7eeb2ff336d/aula/5541868",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 8 times\r\n[in#0 @ 000001c5f50c8e00] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJhOTA0YTI1My1jZGI3LTQ0YmMtOGYyNC04NWY4YmIxMzhlZDUiLCJzZXNzaW9uS2V5IjoiZmNhZHd1Y3EiLCJwcm9kdWNlcktleSI6IjYwNTQ2NCIsInByb2R1Y2VyRW1haWwiOiJhcmlhbmUuY2FtcG9saW1AZ21haWwuY29tIiwiY29uc3VtZXJFbWFpbCI6Im1hcmlhbmFjbmV2ZXNAaG90bWFpbC5jb20iLCJpbXByaW50IjpmYWxzZSwic3RvcmFnZSI6InIyIiwicGwiOiIxMDgwcF9wbGF5bGlzdC5tM3U4IiwiaWF0IjoxNzE2MzQ3NzEzLCJleHAiOjE3MTY1MjA1MTN9.7Tl0StVgPMMY1W2MWhLq1mqpFrj0sjRoNsU4h5TtHsA.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2843695113603739",
    "error": "coapp_failure",
    "id": "coapp_failure_d67b5f4e-b3ef-44b3-b137-1fcefdcbfe7f"
  }
}

Como entro em contato cum este Paulo para resolver meu problema? Seu eu comprar a licença resolvo este problema?

**Reply by GrampaWildWilly** [reply]
You've posted here.  That is all you need to do to contact @paulrouget Paul.  Like I said, he's the developer.  If you have paid any attention at all to other activity in this forum, you'll know Paul is quite active here.  He will come in here & say something eventually.

A license is unlikely to have any effect on this.  The only thing the VDH license does is it skips writing the watermark into your videos.  Nothing else.  VDH otherwise is identical, with & without a license.  Getting a license does not unlock features.  The unlicensed VDH has all the same functions as the licensed VDH.  There are no web sites that become accessible with a license.  If a site works with VDH without a license, it will work with one.  If a site does not work without a license, it still won't work with a license.  Like I said, the license just skips writing the watermark.  Now, writing the watermark is typically a very time consuming operation.  Depending on the size of the video, it could take 6 hours or longer to write the watermark.  So, buying a license is like an act of good citizenship.  You are showing support for the development of VDH.  However, if VDH doesn't work on sites you want to download from, then there's no point in buying a license because you won't be using VDH.  In your case, you never downloaded anything so the step of writing the watermark has not even occurred yet.  Clearly, you can see that buying a license won't have any effect on this problem.

You might want to search this forum for discussions of antivirus & firewalls.  There are several such discussions that offer advice that might help you.  Maybe.  Possibly.  No guarantees.  But it's possible.
